<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmRemoteSessionDialog</class>
 <widget class="QDialog" name="ConfirmRemoteSessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm Remote Session</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CAUTION</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Executing malicious commands on remote hosts can cause serious security damage.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_remote_host">
     <property name="text">
      <string>Do you approve work in the following directory?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ReadOnlyLineEdit" name="lineEdit_remote_host"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_remote_command">
     <property name="text">
      <string>Do you approve of executing the following command?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ReadOnlyLineEdit" name="lineEdit_remote_command"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadOnlyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ReadOnlyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>ConfirmRemoteSessionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConfirmRemoteSessionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
